src/Helpers/LoadingSpinner.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import "./LoadingSpinner.css";
+
+export default function LoadingSpinner() {
+  return (
+    <div className="spinner-container">
+      <div className="loading-spinner"></div>
+    </div>
+  );
+}
```

Comment:
It's up to you but there is a separate package for spinners, `react-spinners`...